proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -127,61 +127,107 @@ get space(): KnownColorSpace;
 
 #### `channels`
 
-Returns an array of channel values for [`internal`], with missing channels
-converted to `0`.
+Returns an array of channel values (excluding `alpha`) for [`internal`], with
+[missing channels][missing components] converted to `0`.
+
+[missing components]: ./color-4-new-spaces.md#missing-components
 
 ```ts
-get channels(): string | number;
+get channels(): [number, number, number];
 ```
 
 #### `channelsOrNull`
 
-Returns an array of channel values for [`internal`], with missing channels
-converted to `null`.
+Returns an array of channel values (excluding `alpha`) for [`internal`], with
+[missing channels][missing components] converted to `null`.
 
 ```ts
-get channelsOrNull(): ChannelValue;
+get channelsOrNull(): [ChannelValue, ChannelValue, ChannelValue];
 ```
 
 #### `isLegacy`
 
-Returns whether [`internal`] is in a legacy color space (`rgb`, `hsl`, or
+Returns whether [`internal`] is in a [legacy color space] (`rgb`, `hsl`, or
 `hwb`).
 
 ```ts
 get isLegacy(): boolean;
 ```
 
+[legacy color space]: ./color-4-new-spaces.md#legacy-color
+
 #### `isInGamut`
 
-Returns whether [`internal`] is in-gamut for its color space (as opposed to
+Returns whether [`internal`] is [in-gamut] for its color space (as opposed to
 having one or more of its channels out of bounds, like `rgb(300 0 0)`).
```

Comment:
Nit: it's fine to have a non-normative clarifying note here, but for consistency with other specs it should be formatted as a block quote.

Reply:
This was removed in a refactor in 422af09eb7ec627a10178e07df803f68b6130e80